src/components/Table/Table/__stories__/helpers.js

```diff
@@ -32,16 +82,127 @@ export const emailColumns = [
   {
     id: "sentOn",
     title: "Sent on",
-    width: 150
+    width: 150,
+    contentSkeleton: "long-text"
```

Comment:
this can't be seen in the story, which makes it hard to understand how to use
![image](https://github.com/mondaycom/monday-ui-react-core/assets/6093192/c4b5fc1b-bbf6-4e3e-965f-16cc97d60f64)

Reply:
hmm I get what you mean.
We need to think of a technical solution for that, maybe add some "assets" feature to stories for cases like this where we want to show api usage.
I'll think of a median solution for now

Reply:
I'd just keep the data inside the render function